data/json/monsters/zed_misc.json

```diff
@@ -89,6 +89,59 @@
     },
     "flags": [ "SEES", "HEARS", "SMELLS", "KEENNOSE", "STUMBLES", "WARM", "POISON", "GRABS", "NO_BREATHE", "FILTHY" ]
   },
+  {
+    "id": "mon_zombullfrog",
+    "type": "MONSTER",
+    "name": { "str": "zombullfrog" },
+    "description": "The wound covered skin of this large bullfrog appears to be peeling in places, revealing a black goop beneath the surface.  Its vacant stare passes by anything that isn't food, and it will leap to devour anything it can.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "120 L",
+    "weight": "150 kg",
+    "hp": 120,
```

Comment:
They're twice the size of a zombie, they could stand to have at least twice as much health (especially if their armor is this low).

Reply:
120 was still a big upgrade from mutant bullfrog (46) and giant bullfrog (70), but yeah, it's poorly armored, so it could get some more HP. Bumping it to 160.